tools/test/audio/process_test.m

```diff
@@ -357,8 +359,8 @@
 % Test constraints
 test.f_start = 20;
 test.f_end = test.fs * 0.41667; % 20 kHz @ 48 kHz
-test.fu = test.fs * 0.41667;    % 20 kHz @ 48 kHz
-test.f_max = 0.999*t.fs/2;      % Measure up to min. Nyquist frequency
+test.fu = test.fs * 0.41667;    % 20 kHz @ 48 kHz, duplicate variable/ instead use test.f_end
+test.f_max = 0.999*test.fs/2;   % Measure up to min. Nyquist frequency
```

Comment:
I was recalling this had to do something with SRC/ASRC multi-rate testing. Can you please check the impact to src_test.m. This would be best to be in separate commit with change description.

Reply:
I'll make a fresh pull request and submit the modifications for the sample rate test.